.github/workflows/ci-test-docs-generation.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.19.x # TODO(friel) use version_set
+          go-version: 1.20.x # TODO(friel) use version_set
```

Comment:
Any chance we could address this TODO since we're touching this line and not much else? WDYT?

Reply:
Will explore

Reply:
TY!

Reply:
Trying out in https://github.com/pulumi/pulumi/pull/12048. Not ready for review yet.